src/components/questions/answer.js

```diff
@@ -1,12 +1,87 @@
 import React from "react";
-import formatDate from "../../formatDate"
+import formatDate from "../../formatDate";
+import axios from "axios";
 
 class Answer extends React.Component{
 constructor(props){
   super(props);
-  this.state = {};
+  this.state = {
+    isReported:false
+  };
+  this.helpfulClick = this.helpfulClick.bind(this);
+  this.sendHelpful = this.sendHelpful.bind(this);
+  this.reportedClick = this.reportedClick.bind(this);
+  this.sendReported = this.sendReported.bind(this);
 }
 
+//INITIALIZES LOCAL STORAGE OBJECTS FOR ANSWERS ON MOUNT
+componentDidMount(){
+  localStorage.setItem("helpfulAnswers",JSON.stringify([]));
+  localStorage.setItem("reportedAnswers",JSON.stringify([]));
+}
```

Comment:
Won't this wipe your local storage items every time the component mounts?

Reply:
totally correct! i forgot to get back to this. will adjust now.